src/components/Map/__tests__/Map.spec.js

```diff
@@ -82,4 +86,35 @@ describe('<Map/> that renders a Map with markers', () => {
         expect(setZoomMock).toHaveBeenCalledTimes(1);
         expect(setCenterMock).toHaveBeenCalledTimes(1);
     });
+    it('should add highlighted areas when geoJson objects are passed as markers', () => {
+        mount(<Map markers={[DR_DRENTHE, FL_FLEVOLAND, FR_FRIESLAND, GE_GELDERLAND]} />);
+        expect(addGeoJsonMock).toHaveBeenCalledTimes(4);
+        expect(setZoomMock).toHaveBeenCalledTimes(1);
+        expect(setCenterMock).toHaveBeenCalledTimes(1);
+    });
+    it('should recenter the map and change zoom when geoJson objects are replaced by regular markers', () => {
+        const wrapper = mount(
+            <Map markers={[DR_DRENTHE, FL_FLEVOLAND, FR_FRIESLAND, GE_GELDERLAND]} />
+        );
+        expect(setZoomMock).toHaveBeenCalledTimes(1);
+        expect(setCenterMock).toHaveBeenCalledTimes(1);
+        wrapper.setProps({ markers: [pointMarker, regionMarker] });
+        expect(setZoomMock).toHaveBeenCalledTimes(2);
+        expect(setCenterMock).toHaveBeenCalledTimes(2);
+    });
+    it('should center map when geoJson objects are used as markers', () => {
```

Comment:
this is (almost) same as test in line 89. This one can be removed

Reply:
Removed.